test/unit/test_configparser.py

```diff
@@ -0,0 +1,208 @@
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+
+from __future__ import annotations
+
+from pathlib import Path
+from test.randomize import random_string
+from textwrap import dedent
+from typing import Callable, Dict, Union
+
+import pytest
+from pytest import raises
+
+try:
+    from snowflake.connector.config_parser import ConfigParser
```

Comment:
Should we name it `SnowflakeConfigParser` to avoid confusion with `configparser.ConfigParser`?

Reply:
I'm not a fan of prepending Snowflake to anything anymore. I'm not against it, but if others don't feel like we need this I'd rather not

Reply:
After some discussion with my team, we've decided not to do this.
If it's necessary for some user we believe that `import-as` statements should solve their issues.